src/app/features/users-page/interfaces/users.ts

```diff
@@ -1,4 +1,4 @@
-export interface IUsers {
+export interface Users {
```

Comment:
I'm pretty sure this interface represent single user not users

`User[]` means users;
`Users[]` means array of another collection of users